test/pytest/test_utils.py

```diff
@@ -59,6 +59,8 @@ def start_torchserve(
     no_config_snapshots=False,
     gen_mar=True,
     plugin_folder=None,
+    models=None,
+    mode=None,
```

Comment:
The default mode is called "none" in the docs. But here None selects explicit. Better to revert the logic to no confuse people.

Reply:
The logic for test is to keep enabled as default so that it does not break existing test structure for now. Will require complete test overhaul at a later date.

Reply:
You can just set the default value to model the old behavior to not break any test.

Reply:
We want TorchServe default to be model api enabled as false meaning that users will not be able to register or delete models once it starts. That change will break the tests

Reply:
Udai, this is not about the way people will setup TorchServe, this is only about the interface of start_torchserve which is not user facing.